tests/fast/CycleTest.toml

```diff
@@ -6,35 +6,3 @@ testTitle = 'Clogged'
     transactionsPerSecond = 2500.0
     testDuration = 10.0
     expectedRate = 0
-
```

Comment:
Is it intentional to remove all the failure injection here?

Reply:
yes and no -- I wanted to change this to make the test much faster (I don't need failure injection yet to test authorization). But I'll need to revert this before making the PR a non-draft